app/components/election-time-input.js

```diff
@@ -0,0 +1,58 @@
+// https://github.com/EnCiv/undebate-ssp/issues/6
+
+import { useEffect, useRef, useState } from "react"
+import { createUseStyles } from "react-jss"
+import svg from "../../assets/svg/clock.svg"
+
+const ElectionTimeInput = props => {
+    const { defaultValue = "", onDone = () => {} } = props
+    const [time, setTime] = useState(defaultValue)
+    const classes = useStyles(time)
+    const inputRef = useRef(null)
+
+    useEffect(() => {
+        handleDone(time)
+    }, [])
+
+    const handleChange = e => {
+        setTime(e.target.value)
+    }
+
+    const handleDone = e => {
+        onDone({ valid: time ? true : false, value: time })
+    }
+
+    return (
+        <>
+            <input
+                className={classes.electionTimeInput}
+                type="time"
+                defaultValue={time}
+                onBlur={handleDone}
+                onChange={handleChange}
+                onKeyPress={e => {
+                    if (e.key === "Enter") inputRef.current.blur()
+                }}
+                ref={inputRef}
+                placeholder=""
+            />
+        </>
+    )
+}
+
+export default ElectionTimeInput
+
+const useStyles = createUseStyles({
+    electionTimeInput: time => ({
+        borderRadius: "0.625rem",
+        background: "linear-gradient(0deg, rgba(38, 45, 51, 0.2), rgba(38, 45, 51, 0.2)), #FFFFFF",
+        color: time ? "black" : "grey",
+        padding: "1rem 1.25rem",
+        border: "none",
+        fontSize: "1.125rem",
+        width: "100%",
+        "&::-webkit-calendar-picker-indicator": {
```

Comment:
You have been able to do so much with just the one input field. But here is a challenge - classes that begin with -webkit are specific to the chrome.  This works on chrome. But does not work on safari or firefox.  We need to make our UI work with any browser - so we have to either find a standard way to do this, or find the equivalent class names for safari and firefox.  But in general find the standard way. 

Another challenge is that on Saturday's hackathon someone else added a clock.svg file.  And the two are slightly different.  -- the other one has some transparency to it - and both are different grid boxes.  

So I suggest you do:
```
mv assets/svg/clock.svg assets/svg/clock-solid.svg
git fetch
git checkout main
git pull
git checkout election-time-input#6
git merge main
```
But I wish there was some way to have the one clock.svg and both use it but modify as needed.  But whatever works is fine.

Reply:
Ok, I'll resolve the clock.svg issue. In terms of the webkit compatibility I wasn't able to find a standard, cross-browser compatible way to customize the icon but I will look at ways to make it compatible with Safari and Firefox and make sure there is a decent fallback for any other browsers.